armi/reactor/converters/tests/test_axialExpansionChanger.py

```diff
@@ -109,7 +109,7 @@ def _storeTargetComponentMassAndDensity(self, c):
         tmp = array(
             [
                 c.getMass(),
-                c.material.getProperty("density", c.temperatureInK),
+                c.material.getProperty("pseudoDensity", c.temperatureInK),
```

Comment:
@albeanth This looks a bit odd to me.  Is this the way you want it?

Reply:
It should be `density3` I think. But the density portion of that array looks like it's unused in the tests... So it could probably get removed.

Reply:
Er, in the case of this PR, not `density3`, but rather just `density`.

Reply:
Thanks, @albeanth !  Fixed!